jobs/import-to-table-partition.yml

```diff
@@ -31,6 +32,11 @@ configuration:
       scheme: "psSite"
       column: "partition_num"
       value: "${partitionNum}"
+      incremental:
+        enabled: "${incrementalEnabled}"
+        newWatermarkQuery: "sql/incremental/newWatermarkQuery.sql"
+        previousWatermarkQuery: "sql/incremental/previousWatermarkQuery.sql"
+        deleteQuery: "sql/incremental/deleteQuery.sql"
     bulkCopy:

```

Comment:
After our recent discussion @dmdesimone , I realized it would be best to try to embed this capability in the existing SQL Server Bulk import job, rather than making it all config driven or making a separate job type.  There is a lot of existing functionality to reuse, and it made sense here when I implemented it.

For more details on how this is intended to work, you can see my commit/PR here:  https://github.com/PIH/petl/pull/32

In an ideal world, much like we have now, we would configure all of this once like the above with these conventions, and the only thing that would need to vary from job to job is the schema (to include the necessary columns that are expected) and the extract (to load the necessary columns and limit the data being queried appropriately).  But these watermark and delete queries would always be the same.